cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authorization/AccessController.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.security.spi.authorization;
```

Comment:
nit: Should we maybe consider reorganizing the original and new access control layers into `v1` and `v2` subpackages respectively? It may be hard for developers to understand the relationship between the AccessController and the Authorizer even though the Access* layer is meant to replace the Authorization* layer 1-1.

Reply:
I don't want to introduce V2 as V1 will go away hopefully within a yer or two and I don't want to end up with V2 only

Reply:
Acknowledged, SG, thanks!